### ISSUE TITLE
numOfClauses and numOfCycles

### ISSUE THREAD

**Issue by rodrigoazs** [hayesall/srlearn#75]
**Describe the bug**
There is a bug when defining a background. The params used by BoostSRL are numOfClauses and numOfCycles, however the code is generating as numberOfClauses and numberOfCycles.

**System information**
In the background.py

```bash
            "number_of_clauses": "setParam: numberOfClauses={0}.\n",
            "number_of_cycles": "setParam: numberOfCycles={0}.\n",
```

it should be:
```bash
            "number_of_clauses": "setParam: numOfClauses={0}.\n",
            "number_of_cycles": "setParam: numOfCycles={0}.\n",
```

**To Reproduce**
The current code is not able to set a max number of clauses and cycles.

**Expected behavior**
..

**Screenshots**
..

**Additional context**
..

**Comment by hayesall** [comment]
Thanks for catching this! Should be fixed pretty soon.

**Comment by hayesall** [comment]
Also @rodrigoazs I read your transfer learning paper a while back.

Do you have preferences for editing the dependency networks?

I was hacking on an internal representation thinking the objects could be produced/consumed (sort of like the plot/export digraph examples in the [Family Relationships Example](https://srlearn.readthedocs.io/en/latest/auto_examples/plot_families.html#sphx-glr-auto-examples-plot-families-py))

Something like:

```lisp
rewrite(clf, "advised_by", "worked_under")
```

**Comment by rodrigoazs** [comment]
Hi @hayesall, thanks for reading it!

I think that sounds very helpful so that we can have a new classifier object with the predicates changed (in order to do that I had to parse the files and change it). But in my case, to transfer dependency networks, I had to relearn the parameters in order to fit the dependency networks to the new data. I think that still requires a modification in the BoostSRL.
